package/yast2-users.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu May  5 12:32:39 UTC 2016 - igonzalezsosa@suse.com
+
+- Fix users import when an entry in /etc/shadow is missing
```

Comment:
I've updated the message but using `non-system` instead of `local` (although we use the term `local` in the code) :)